src/main/include/RobotContainer.h

```diff
@@ -8,18 +8,18 @@
 
 #include "T34CommandXboxController.h"
 #include "subsystems/SwerveDrive.h"
+#include "subsystems/Climber.h"
 #include "commands/ControllerDriveCommand.h"
 
 class RobotContainer {
  public:
   RobotContainer();
 
   static RobotContainer* Get();
-
   std::shared_ptr<t34::SwerveDrive> swerve_drive;
   std::shared_ptr<t34::T34CommandXboxController> ctrl;
-
   t34::ControllerDriveCommand DefaultCommand;
+  Climber climber;
```

Comment:
`climber` and `DefaultCommand` should probably be private.

If you look at [RapidReactCommandBot.h][RapidReactCommandBot], the equivalent of `RobotContainer` in the example `RapidReactCommandBot` project (_I don't know why they broke from the tradition of naming it `RobotContainer` 🤷‍♂️_), you'll see that they make all their subsystems private in `RapidReactCommandBot`.

Additionally, per our naming convention …

```suggestion
  t34::ControllerDriveCommand m_default_command;
  Climber m_climber;
```

[RapidReactCommandBot]: https://github.com/wpilibsuite/allwpilib/blob/main/wpilibcExamples/src/main/cpp/examples/RapidReactCommandBot/include/RapidReactCommandBot.h

Reply:
updated to m_climber/m_default_command